roles/as_code/lib/engineer.rb

```diff
@@ -1,16 +1,45 @@
 require_relative './shared_expectations/core_engineer_expectations'
 
 JobSpec::Role.definition 'Engineer' do
-  description 'Our software delivery teams are the core of our business. They are usually comprised of a Lead Engineer, a Delivery Manager, and one or more Engineers. As an Engineer we expect someone:'
+  description <<~DESCRIPTION
+    Our engineers build software that make our customers happy, prefer problem solving over task completing, love learning from colleagues, and constantly strive to be nice humans.
+
+    ## What does the job entail?
+
+    We primarily write and deliver custom software to our customers. We build solutions in a variety of languages and platforms, historically Ruby on Rails, we tend to use React, JavaScript, Elixir, and Clojure these days. We will usually be responsible for setting up a customer’s infrastructure either on AWS with tools like Ansible and Terraform. Sometimes we opt for a Platform as a Service like Heroku. We will make sure there is a continuous delivery pipeline running our automated test suites before deploying to a staging environment, and a manual button for going to production.
+
+    The software we build solves business problems. We’ve built software for karaoke, e-commerce, retail buying departments, accountants, photography studios, warehouses, supply chain companies, and ourselves. Typical projects will last 3-6 months, some customers work with us over longer periods but we like to mix up teams at around 6 months to keep things fresh.
+
+    ## What experience are we looking for?
+
+    We’ll expect you to be proficient in at least two programming languages. This is usually something like PHP/Ruby/Python/Java/C# and then JavaScript. Everyone in the organisation comes from various language backgrounds and we see this as a strength of our offering, our teams can solve problems with open minds and across many platforms. This also makes for a great learning environment.
```

Comment:
IDK if leading with PHP sets the right tone?

Reply:
(not intending to start some sort of language flamewar)

Reply:
lol – you just did